### PR TITLE
Updated to include SSL compatibility

### PR DESCRIPTION
**Pull request by ObsessiveOrange**
Signed-off-by: Benedict Wong <bennydictwong@gmail.com>

### CODE REVIEW

**Comment by ObsessiveOrange** [comment]
Tied to CodeCollaborate/Server/pull/35

File: src/main/java/websocket/WSConnection.java [hunk 1]
```diff
@@ -72,7 +73,8 @@ public void handleEvent(EventType event) {
     public void connect() throws Exception {
         try {
             if (this.client == null) {
-                this.client = new WebSocketClient();
+                SslContextFactory ssl = new SslContextFactory();
```
Comment (julie-is-late) [inline src/main/java/websocket/WSConnection.java:76]: what happens if the server doesn't have ssl enabled?

Comment (ObsessiveOrange) [reply src/main/java/websocket/WSConnection.java:76]: This doesn't do anything then. it just enables ssl if need be

**Comment by julie-is-late** [comment]
LGTM

**Pull request merged**